src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetV1.java

```diff
@@ -493,7 +478,7 @@ public InputStream getBinaryStream(int columnIndex) throws SQLException {
     @Override
     public String getString(String columnLabel) throws SQLException {
       raiseSQLExceptionIfResultSetIsClosed();
-      return null;
+      return "";
```

Comment:
With this change, getString(String columnLabel) matches behavior of getString(int columnIndex). Empty string instead of null seems safer.

Reply:
+1